barretenberg/cpp/src/barretenberg/aztec_ivc/aztec_ivc.hpp

```diff
@@ -62,9 +63,11 @@ class AztecIVC {
         MSGPACK_FIELDS(folding_proof, decider_proof, goblin_proof);
     };
 
+    enum class QUEUE_TYPE { OINK, PG };
     struct FoldingVerifierInputs {
         FoldProof proof;
         std::shared_ptr<VerificationKey> instance_vk;
+        QUEUE_TYPE type;
```

Comment:
Seems like this struct and the proof type inside should be renamed now because it's not about folding in the Oink case, right?

Reply:
Also, why not handle the merge steps using this queue as well?